lib/presentation/login/components/oauth_webview.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:appetizer/data/constants/env_config.dart';
+import 'package:appetizer/utils/app_extensions/app_extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:auto_route/auto_route.dart';
+
+@RoutePage()
+class OAuthWebScreen extends StatelessWidget {
+  static const id = 'oauth_view';
+  OAuthWebScreen({Key? key}) : super(key: key);
+  final ValueNotifier<int> _loadingState = ValueNotifier(1);
+  final String omniportSignUpURL =
+      'https://channeli.in/oauth/authorise/?client_id=${EnvironmentConfig.OAUTH_CLIENT_ID}&redirect_uri=${EnvironmentConfig.OAUTH_REDIRECT_URI}';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: ValueListenableBuilder(
+          valueListenable: _loadingState,
+          builder: (context, value, _) {
+            return IndexedStack(
+              index: int.parse(value.toString()),
```

Comment:
why aren't you passing the value directly?

Reply:
Fixed